test-lit/mm3.mod

```diff
@@ -0,0 +1,4 @@
+﻿!mod$ v1 sum:8d1e01df5c32946e
```

Comment:
I'm not sure how this managed to end up in this commit, I specifically went out of my way to avoid that...Will remove in next patch set.